CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ The format is described in [Contributing notes](CONTRIBUTING.md#changelog-entry-
 
 ## master
 
+### Added
```

Comment:
Don't forget to add the `release` header here, see https://guides.rubygems.org/patterns/#semantic-versioning to choose what's the version you should use for this release